tests/jballerina-unit-test/src/test/resources/test-src/types/typedesc/typedesc_positive.bal

```diff
@@ -1,3 +1,5 @@
+import ballerina/http;
```

Comment:
Instead of importing `http` shall we introduce a new object here? 

Or if the test is for an object from an imported module, shall we use something from langlib or just add a separate balo test?

Also, I guess it would be better to move these tests to TypeGuardTest.java and test type narrowing also?